src/types.ts

```diff
@@ -16,11 +16,13 @@ import { Point as RoughPoint } from "roughjs/bin/geometry";
 import { LinearElementEditor } from "./element/linearElementEditor";
 import { SuggestedBinding } from "./element/binding";
 import { ImportedDataState } from "./data/types";
-import { ExcalidrawImperativeAPI } from "./components/App";
+import App from "./components/App";
```

Comment:
This can lead to circular dependency?

Reply:
```suggestion
import type App from "./components/App";
```

Maybe. So short of factoring those App types out in here, let's at least import it as type only.

Reply:
yeah did the same for `library.ts` as well